src/runtime/components/NuxtImg.vue

```diff
@@ -136,7 +136,10 @@ onMounted(() => {
   if (placeholder.value || props.custom) {
     const img = new Image()
 
-    if (mainSrc.value) {
+    if (placeholder.value) {
+      img.src = placeholder.value
+    }
```

Comment:
Even like this, the placeholder image will be loaded twice in SSR environment: 
- once by the `<link rel="preload" ... />` tag in the head 
- once by the `<img />` tag in the template
However the actual image is loaded once when `placeholderLoaded.value` is set to `true` in the `onload` callback of the placeholder image